services/order.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from typing import List, Dict
+
+User = get_user_model()
+
+
+@transaction.atomic
+def create_order(
+        tickets: List[Dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)

```

Comment:
Consider handling the `DoesNotExist` exception that can be raised by `User.objects.get(username=username)`. This will prevent runtime errors if the username is not found.